packages/iris-grid/src/formatters/DateTimeColumnFormatter.ts

```diff
@@ -128,11 +157,11 @@ class DateTimeColumnFormatter extends TableColumnFormatter {
     );
   }
 
-  getEffectiveFormatString(baseFormatString) {
+  getEffectiveFormatString(baseFormatString: string): string {
     return this.formatStringMap.get(baseFormatString) || baseFormatString;
   }
 
-  format(value, format) {
+  format(value: unknown, format?: TableColumnFormat): string {
```

Comment:
Can this value be set using the generic in `TableColumnFormatter`? Or is it always unknown? If always unknown, the `TableColumnFormatter` class might not need to have a generic type

Reply:
I think always `unknown`. I will revisit when converting grid to TS (#21)